Libraries/dnYara.Interop/Interops/ObjRefHelper.cs

```diff
@@ -126,5 +134,13 @@ public static bool CheckTag(IntPtr tag_ptr, out string tagName)
             return true;
 
         }
+
+        public static readonly int RULE_GFLAGS_NULL = 0x1000;
```

Comment:
Can be put in **Libraries\dnYara.Interop\Interops\Constants.cs**.

Reply:
Done, thanks for the pointer